massdriver-application/gcp_identity.tf

```diff
@@ -3,7 +3,7 @@
 # namespace and k8s service account name
 
 locals {
-  gcp_identity = local.is_kubernetes ? {
+  gcp_identity = local.is_gcp && local.is_kubernetes ? {
```

Comment:
You should do this like the Azure one where the outside block is just `local.is_gcp` and the inner kube one is `local.is_kubernetes`.

Reply:
Ok, I'll add that in